theme/static/js/hs_file_browser.js

```diff
@@ -612,7 +612,7 @@ function bindFileBrowserItemEvents() {
 
             var fileAggType = [];
             // main-file is available on the aggregation folder and only single file aggregations have a data-pk of 1 on the file
-            if ($(event.target).closest("li").attr("main-file") || $(event.target).closest("li").attr("data-pk") === "1"){
```

Comment:
this was a bug, my testing wasn't thorough enough.  I've updated hydroshare to pass the single-file information to the front end instead of trying to make something work with what was available.